language-plutus-core/generators/Language/PlutusCore/Generators/Interesting.hs

```diff
@@ -181,6 +183,31 @@ genNatRoundtrip = do
         return $ mkIterApp () (TyInst () natToInteger size) [ssize, n]
     return . TermOf term $ TypedBuiltinValue typedIntS iv
 
+-- | @sumNat@ as a PLC term.
+getBuiltinNatSum :: Size -> Quote (Term TyName Name ())
+getBuiltinNatSum s = rename =<< do
```

Comment:
Moved this from `Meta.hs` as it's not a standard library combinator, just some weird function that turns a list of natural into a list of integers and adds them up. It also receives a `Size`, because when it was written there were no handy combinators to deal with sized types, so this should be fixed. later